public/views/address.html

```diff
@@ -37,7 +37,7 @@
                 <td><strong>Address</strong></td>
                 <td class="text-right"></span>{{address.address}}&nbsp;<span class="btn-copy" clip-copy="address.address"></span></td>
               </tr>
-              <tr class="hidden-xs">
+              <tr class="hidden-xs" data-ng-if="!(address.publicKey >= 0)">
```

Comment:
`data-ng-if="address.publicKey"` is not enought? We shouldn't compare strings such way.